Stratis.Bitcoin/Features/Wallet/Helpers/SendTransactionComparer.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Stratis.Bitcoin.Features.Wallet.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+namespace Stratis.Bitcoin.Features.Wallet.Helpers
+{
+    /// <summary>
+    ///  Used for comparing a list of TransactionItemModel to each other.
+    /// </summary>
+    /// <remarks>
+    ///  This specific comparer has been made to filter out duplicate payment listings in the case of a transaction with multiple inputs.
+    ///  This causes multiple items inside the wallet data that all have the same amount, transaction/spending id and payment details.
+    ///  However not all fields on this model are equal to each other so we only compare the ones we need.
+    ///  See: https://github.com/stratisproject/Breeze/issues/175
+    /// </remarks>
+    public class SendTransactionComparer : IEqualityComparer<TransactionItemModel>
```

Comment:
maybe call this either Sen_t_TransactionComparer or TransactionItemModelComparer?